how-to/workspace-platform-starter/client/src/framework/platform/platform-mapper.ts

```diff
@@ -576,7 +668,7 @@ function platformLayoutComponentToStorage(component: ComponentType): void {
 				{ propName: "uuid", defaultValue: "" },
 				{ propName: "componentName", defaultValue: "view" },
 				{ propName: "initialUrl", defaultValue: "" },
-				{ propName: "processAffinity", defaultValue: "" },
+				{ propName: "processAffinity", defaultValue: fin.me.identity.uuid },
```

Comment:
I think this might be tricky. Is there a danger that we put the uuid as the default when it shouldn't be? The platform can have different affinty strategies (different ...everything gets a unique affinity or same). Is there a risk we mess with that (as process affinity is something that people care about a lot)?

Reply:
It will only remove from payload if its set to fin.me.identity.uuid